src/endpoint/endpoint.js

```diff
@@ -58,10 +58,21 @@ function start_all() {
     if (cluster.isMaster &&
         config.ENDPOINT_FORKS_ENABLED &&
         argv.address &&
-        !argv.s3_agent &&
-        process.env.container !== 'docker') {
+        !argv.s3_agent) {
         // Fork workers
-        const NUM_OF_FORKS = (os.totalmem() >= (config.SERVER_MIN_REQUIREMENTS.RAM_GB * size_utils.GIGABYTE)) ? numCPUs : 1;
+        const FORKS_ACCORDING_TO_MEM = Math.max(
+            1,
+            Math.floor((os.totalmem() - (4 * size_utils.GIGABYTE)) / 2)
```

Comment:
I think you need to divide by 2GB, not by 2, right?
Perhaps it would be simpler to just make this entire calculation in MB like this:
```
const total_mem_mb = Math.floor(os.totalmem() / 1024 / 1024);
const reserved_mem_mb = 4096;
const fork_mem_mb = 2048;
const FORKS_ACCORDING_TO_MEM = Math.max(1, Math.floor((total_mem_mb - reserved_mem_mb) / fork_mem_mb));
```